readme.md

```diff
@@ -110,6 +110,7 @@
 - [Web Accessibility](https://github.com/brunopulis/awesome-a11y)
 - [Material Design](https://github.com/sachin1092/awesome-material)
 - [CSS](https://github.com/sota0805/awesome-css)
+- [Gulp](https://github.com/alferov/awesome-gulp)
```

Comment:
Agree. I had that doubts, because it's more about workflow automation.
Developer Environment then?
Anyway, most of the gulp work is to automate frontend assets management, that's why I added it under Front-end Development